coursedata/texts/en.yaml

```diff
@@ -66,6 +66,7 @@ HedyErrorMessages:
     Invalid Argument Type: "You cannot use {command} with {invalid_type}{invalid_argument}. Try using {command} with: {allowed_types}."
     Unsupported Float: "Unfortunately, Hedy does not support non-integer numbers. Try changing {value} to an integer."
     Locked Language Feature: "You are using {concept}! That is awesome, but {concept} is not unlocked yet! It will be unlocked in a later level."
+    Invalid Argument: "{arg} is not a valid argument for the {command} command."
```

Comment:
```suggestion
    Invalid Argument: "{arg} is not a valid argument for {command}."
```

Reply:
I prefer to just refer to the command, but also I think this is a complex error message for a kid. What is an argument? Why is it not valid?

Reply:
I removed this error message. Instead I switched it's only remaining use to UndefinedVarException.